client/dist/index.html

```diff
@@ -1,5 +1,10 @@
 <head>
-  <link rel="stylesheet" type="text/css" href="./graph.css">
+  <link rel="stylesheet" type="text/css" href="/graph.css">
```

Comment:
Be careful when using root route - it might have unintended behavior in some cases. If your graph.css is in the same folder as your index.html, `href=graph.css` would achieve the desired effect. (Unless you already tried it and it didn't work, in which case ignore this comment)

Reply:
Yeah - this might be a side-effect of specifying the /game portion of the URL - relative paths are relative to /graph, and my router points all references to /graph... to my index.html - to make it work, I would need to carve out other router-specified directories to point supporting files to... which wouldn't be such a bad idea, now that I think about it. 

I really just wish that putting my files into /client/dist/game would work like you'd expect it to. Maybe I'll check in with a Help Desk person to see why it's not.